glTF-BinExporter/RhinoDocGltfConverter.cs

```diff
@@ -221,6 +277,10 @@ public Rhino.Geometry.Mesh[] GetMeshes(RhinoObject rhinoObject)
                     mesh.EnsurePrivateCopy();
                     validMeshes.Add(mesh);
                 }
+                else
+                {
+                    validMeshes.Add(null);
```

Comment:
Why are we adding a null mesh to the array here?

Reply:
That is so the materials array and the meshes array match. I have a continue; somewhere else to deal with the null mesh. I am open to other solutions though!